sdk/ml/azure-ai-ml/azure/ai/ml/entities/_registry/identity.py

```diff
@@ -0,0 +1,41 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+from typing import Dict, Optional, Union
+
+from azure.ai.ml._restclient.v2022_10_01_preview.models import ManagedServiceIdentity as RestManagedServiceIdentity
+from azure.ai.ml._restclient.v2022_10_01_preview.models import UserAssignedIdentity as RestUserAssignedIdentity
+from azure.ai.ml.constants._workspace import ManagedServiceIdentityType
+
+
+class ManagedServiceIdentity:
```

Comment:
Have you considered reusing entities/_compute/_identity.py?

Reply:
I don't think this will work since we want to solely use system assigned identities while compute uses user assigned identities.

Reply:
@luigiw I reused the IdentityConfiguration class from entities/_credentials. In order to do so, I added to and from rest object methods that matched the 10-01 rest client to the IdentityConfiguration class.